tapdb/sqlc/queries/metadata.sql

```diff
@@ -0,0 +1,2 @@
+-- name: AssetsDBSize :one
+SELECT pg_catalog.pg_database_size(current_database()) AS size;
```

Comment:
Is this query precise enough? What if the database is multi tenant and `tapdb` is just one of the users?

Reply:
Given that it's built-in I trust it to a certain degree. It seems to be accounting for every byte at the time of execution, including unused allocated space,  data marked as deleted etc

The query should return the same size for the db regardless of the user and what they have access to within the db

Reply:
What I mean is that if we're also running `lnd` in the same postgres DB, will it return the total size, or just the size of the DB w/ the attached user?

Reply:
`current_database()` would refer to the database/schema you're using. Assuming there would be a different one for `lnd` and `taproot-assets`, I think this should work.

Reply:
Btw, I tested this locally where I have a single development timescale/postgres instance with different databases.
And I get distinct values for each of the following queries:

```
SELECT pg_catalog.pg_database_size('taprootassets') AS size;
--> 16802671

SELECT pg_catalog.pg_database_size('loop') AS size;
--> 8135535

SELECT pg_catalog.pg_database_size('pool') AS size;
--> 8135535

select current_database();
--> taprootassets
```